cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/BoardsAndWidgetsController.kt

```diff
@@ -13,30 +16,44 @@ class BoardsAndWidgetsController : AbstractVerticle() {
     private lateinit var sender: ConfirmationSender
 
     override fun start() {
-        boardsConfigService = BoardsConfigService()
+        val contentRepository = ContentRepository()
+        boardsConfigService = BoardsConfigService(
+            VolumeStorageFactory.boards(),
+            contentRepository,
+            EntityCleanupHelper(vertx))
+
         val allWidgets = boardsConfigService.getAllWidgets()
         sender = ConfirmationSender(vertx)
-        widgetRuntimeService = WidgetRuntimeService(vertx).init(allWidgets)
+        widgetRuntimeService =
```

Comment:
you could make oneliner from this

Reply:
I use a lot of `google-java-format` autoformatter, so I'm used to things like this. To be honest, oneliners like you propose don't look right to me anymore ☹️